workflows/metagenomics.nf

```diff
@@ -57,7 +58,7 @@ include { CUSTOM_DUMPSOFTWAREVERSIONS } from '../modules/nf-core/custom/dumpsoft
 */
 
 // Info required for completion email and summary
-def multiqc_report = []
+// def multiqc_report = []
 
```

Comment:
Convert all parameters to channel before it enters the workflow

Reply:
Done:

```
    input_ch = Channel.fromPath(params.input)
    INPUT_CHECK ( input_ch )
```